app/views/pages/_editor_guide_text.html.erb

```diff
@@ -55,11 +55,11 @@
     </p>
     <h3><strong><%= community_name %> Article/Post Embed</strong></h3>
     <p>All you need is the full link of the article:</p>
-    <code>{% link https://dev.to/kazz/boost-your-productivity-using-markdown-1be %}</code>
+    <code><%= "{% link #{ApplicationConfig['APP_PROTOCOL']}#{ApplicationConfig['APP_DOMAIN']}/kazz/boost-your-productivity-using-markdown-1be %}" %></code>
```

Comment:
you can use the `app_url` helper to avoid using the app config directly and joining strings: https://github.com/forem/forem/blob/8b60a1873545c125b258e4a1d433598e8617ebb9/app/helpers/application_helper.rb#L214

which calls 

https://github.com/forem/forem/blob/8b60a1873545c125b258e4a1d433598e8617ebb9/app/lib/url.rb#L13